### DISCUSSION TITLE
Edit track config

### DISCUSSION THREAD

**Discussion by MehdiiYac** [GMOD/jbrowse#1634] [Q&A] [answered]
Hello,
I want to modify the track config file (i want to modify the parameters in the following screenshot)
![image](https://user-images.githubusercontent.com/91090403/218507856-9d62d419-1096-4376-b921-e0e6d52fdeb3.png)
Where can i find this file.

Thanks

**Comment by cmdcolin** [comment, marked as answer]
see https://jbrowse.org/docs/configuration_file_formats.html

the screen you are seeing is essentially the data from your tracks.conf in a trackList.json format with some default values added. you can add them into your tracks.conf or convert to trackList.json and paste that block

**Reply by MehdiiYac** [reply]
thank you for your help!
